lutris/gui/lutriswindow.py

```diff
@@ -1198,7 +1202,7 @@ def on_game_activated(self, _view, game_id):
         if game_id:
             game = Game(game_id)
             if game.is_installed:
-                game.launch(launch_ui_delegate=self)
+                game.game_launcher.launch(launch_ui_delegate=self)
```

Comment:
Here's another place to create a launcher. We do launch in a bunch of places, don't we?